sota-implementations/multiagent/iql.py

```diff
@@ -91,7 +91,7 @@ def train(cfg: "DictConfig"):  # noqa: F821
             ("agents", "action_value"),
             ("agents", "chosen_action_value"),
         ],
-        spec=env.unbatched_action_spec,
+        spec=env.single_action_spec,
```

Comment:
```suggestion
        spec=env.single_full_action_spec,
```
Wherever `unbatched_action_spec` is used, it should be changed to `single_full_action_spec`

This is valid for all the places here. As the components that were receiving `unbatched_` are expecting a composite